docs/index.md

```diff
@@ -42,53 +42,169 @@ Start with the [tutorial](/tutorial/) and then check out the [concepts](/concept
 [Concepts](/concepts){ .md-button .main-button--secondary .full } [Tutorial](/tutorial/){ .md-button .md-button--primary .main-button--primary .full }  [Guides](guides){ .md-button .main-button--secondary .full }
 
 <div style="height: 10px"></div>
-<p>Alternatively, read on for a quick sample of Prefect in action.</p>
+<p>Alternatively, read on for a quickstart of Prefect. </p>
 ---
 
-## Quick start: Hello Prefect
+## Prefect Quickstart
 
-Install Prefect with 
+### Step 1: [Install Prefect](/getting-started/installation/)
 
 <div class="terminal">
 ```bash
 pip install -U prefect
 ```
 </div>
 
-See the [install guide](/getting-started/installation/) for more detailed installation instructions.
 
```

Comment:
```suggestion
See the [install guide](/getting-started/installation/) for more detailed installation instructions.
```

I would leave this link in case a user has any issues with install.